contract/magink_wizard/lib.rs

```diff
@@ -0,0 +1,61 @@
+#![cfg_attr(not(feature = "std"), no_std, no_main)]
+
+#[openbrush::implementation(
+    PSP34,
+    PSP34Mintable,
+    Ownable,
+    PSP34Metadata,
+    PSP34Enumerable
+)]
+#[openbrush::contract]
+pub mod magink_wizard {
+    use openbrush::traits::{Storage, String};
+
+    #[ink(storage)]
+    #[derive(Storage, Default)]
+    pub struct MaginkWizardContract {
+        #[storage_field]
+        psp34: psp34::Data,
+        #[storage_field]
+        ownable: ownable::Data,
```

Comment:
why do you have ownable/? where did you use it?
Anyone can call your `mint()` function

Reply:
Ownable should be used as a modifier on the mint function. Already added that implementation